src/openrct2-ui/windows/TrackDesignManage.cpp

```diff
@@ -42,7 +44,7 @@ static rct_widget window_track_manage_widgets[] = {
 };
 
 static rct_widget window_track_delete_prompt_widgets[] = {
-    WINDOW_SHIM(WINDOW_TITLE, WW, WH),
+    WINDOW_SHIM(WINDOW_TITLE, WW2, WH2),
```

Comment:
By using STR_STRING as the Window Title it means you have to set the common format args in the paint function as you have done on L249. But as you are using a fixed string that doesn't change you dont need to do that. Instead just use STR_DELETE_FILE for the Window Title and it will be processed correctly and you don't need to mess about with setting the common format args.

Reply:
Thank you. I'll keep that in mind. I've fixed it now.